react/features/toolbox/components/web/Toolbox.js

```diff
@@ -126,6 +128,11 @@ type Props = {
      */
     _fullScreen: boolean,
 
+    /**
+     * Whether or not the prejoin page is enabled.
+     */
+    _prejoinPageEnabled: boolean,
```

Comment:
Please drop this prop and check. The existence of the pre-join sreen shouldn't determine if we show or not these new buttons. We want to sshow them at all times.